tests/util/server/src/servers/registry.rs

```diff
@@ -120,12 +120,40 @@ async fn registry_server_handler(
     let res = Response::new(UnsyncBoxBody::new(Full::from(body)));
     return Ok(res);
   } else if path.starts_with("/api/scopes/") {
-    let body = serde_json::to_string_pretty(&json!({
-      "id": "sdfwqer-sffg-qwerasdf",
-      "status": "success",
-      "error": null
-    }))
-    .unwrap();
+    let json_data = if path == "/api/scopes/denotest/packages/add" {
```

Comment:
Why don't we just use the `@scope/pkg/meta.json` file? Then we could remove all this and we already have a struct for that that's used in the lsp from deno_graph.